vm/devices/tpm/src/tpm_helper.rs

```diff
@@ -730,17 +740,29 @@ impl TpmEngineHelper {
                             // boot-time AK cert request fails.
                             tracing::info!("Preserve previous AK cert across boot");
 
-                            self.nv_write(write_auth_handle, auth, TPM_NV_INDEX_AIK_CERT, &cert)
-                                .map_err(|error| TpmHelperError::TpmCommandError {
-                                    command_debug_info: CommandDebugInfo {
-                                        command_code: CommandCodeEnum::NV_Write,
-                                        auth_handle: Some(ReservedHandle(
-                                            TPM_NV_INDEX_AIK_CERT.into(),
-                                        )),
-                                        nv_index: Some(TPM_NV_INDEX_AIK_CERT),
-                                    },
-                                    error,
-                                })?;
+                            let res = self.nv_write(
+                                write_auth_handle,
+                                auth,
+                                TPM_NV_INDEX_AIK_CERT,
+                                &cert,
+                            );
+                            tracing::info!(
+                                CVM_ALLOWED,
+                                op_type = "NvWrite",
+                                nv_index = TPM_NV_INDEX_AIK_CERT,
+                                data_size = cert.len(),
+                                success = res.is_ok(),
+                                "Wrote TPM NVRAM index"
```

Comment:
Should we say this is case of writing the previous AK Cert (and remove the tracing above)?